routes/controllers/videoFrame.controller.js

```diff
@@ -0,0 +1,32 @@
+const fs = require("fs");
+const path = require("path");
+
+const convertPath = require("../../services/convertPath");
+const extractFramesFromVideo = require("../../services/extractFramesFromVideo");
+
+exports.videoToFrame = function (req, res, next) {
+  const tempDirectory = path.join(__dirname, "../../temp");
+  const videoTotalPathList = Object.entries(req.body.videoUrlList);
+  const startPointsList = Object.values(req.body.startPoints);
```

Comment:
수진님! 이건 제 개인적인 의견이라 참고만 부탁드립니다! :) 

List 자체가 video Path 전체에 대한 리스트 즉, 나열이라고 생각하여 아래와 같은 변수명도 좋을 것 같습니다.
동일한 맥락으로 `startPoint`도 이미 List이므로 `startPointList`도 좋을 것 같다는 의견입니다!

```suggestion
  const videoPathList = Object.entries(req.body.videoUrlList);
  const startPointList = Object.values(req.body.startPoints);
```